newsletter-august-2022.rst

```diff
@@ -0,0 +1,78 @@
+
+New features
+------------
+
+We've continued building a number of features and bug fixes in our roadmap:
+
+- We have more **Example projects**, which allow users to `get started quickly`_ with our products. We shipped an example for Jupyter Book this month, which is of growing interest to our scientific users.
+- Scientific users are important for us, so we've put together a `landing page <https://docs.readthedocs.io/en/latest/science.html>`_ that highlights the benefits that our scientific users are seeing. 
+- We created a `GitHub action <https://github.com/readthedocs/actions>`_ that allows users to have a link to the Pull Request preview automatically added to the PR description. If users find this useful, we will continue to expand this functionality, and perhaps add it to our core platform so no configuration is needed. 
+- We improved the security and safety of our platform by making sure all invites for a team or project have to be approved by the user.
+
+
+You can always see the latest changes to our platforms in our :doc:`Read the Docs Changelog <readthedocs:changelog>`.
+
+.. _get started quickly: https://docs.readthedocs.io/en/latest/examples.html
+.. _flyout menu: https://docs.readthedocs.io/en/latest/flyout-menu.html
+
+Awesome documentation projects
+------------------------------
+
+We are collecting entries for `Awesome Read the Docs Projects`_ - please do tell us about your favorites, either by sending an email or `opening a Pull Request`_!
+
+-  `Weblate <https://docs.weblate.org/en/latest/>`__ - Weblate is a
+   translation platform with a large documentation project with many
+   translations and customized Read the Docs theme. Documentation aimed
+   at all segments: users, administrators and developers. Also features
+   an extensive Changelog. 
+-  `TomoBank <https://tomobank.readthedocs.io/>`__ - a big list of
+   tomographic datasets and phantoms, featuring especially tables and
+   images and maintained by science community.
+-  `Uberspace <https://manual.uberspace.de/>`__ - Customized sidebar and
+   footer, adding project's branding through custom CSS and HTML to
+   ``sphinx_rtd_theme``. Latest version and release date on front page.
+
+.. _Awesome Read the Docs Projects: https://github.com/readthedocs-examples/.github/
+.. _opening a Pull Request: https://github.com/readthedocs-examples/.github/blob/main/contributing.md
+
+Upcoming features
+-----------------
+
+- We are working on some improvements to our URL handling code, which will allow us to support more flexible URL configurations for projects.
+- As mentioned before, we are shifting some of our focus to frontend & marketing these next few months. We're getting close to shipping our new landing page, which we're excited about.
+- We're continuing to focus on outreach for our new build customization features, so that we can continue to improve them with your feedback.
```

Comment:
```suggestion
- We're continuing to focus on outreach for our new build customization features, so that we can continue to improve them with your feedback.
- Our main theme `sphinx_rtd_theme` will soon be revived after a period of inactivity and we will do a number of smaller releases in Q3 and Q4.
```